locksmithing.lic

```diff
@@ -77,7 +77,7 @@ class Locksmithing
       start_time = Time.now
       wait_for_script_to_complete('pick')
       if Time.now - start_time > 15
-        wait_for_script_to_complete('sell-loot')
+        wait_for_script_to_complete('sell-loot') if sell_loot
```

Comment:
❓ Should this be `settings.sell_loot`? I don't see where `sell_loot` is defined

Reply:
Fixed!